src/client/game/game-reducer.js

```diff
@@ -134,6 +135,12 @@ const game = (state = initialState, action) => {
                 disableVowel: false
             })
         }
+		case actionTypes.GAME_PARAMETERS:{
```

Comment:
This file should be tested really - the important thing to test is that the state returned has a different object reference than the one passed in. Maybe hold off on this/ask the PO